src/PlotReducer.tsx

```diff
@@ -0,0 +1,37 @@
+import { ReducerActions, State } from './types';
```

Comment:
by convention this file should be called PlotReducer.ts, tsx is for jsx with typescript

Reply:
> At the end of the blue curve you have a curve, is this supposed to be correct?
> <img alt="Screenshot 2021-01-20 100651" width="541" src="https://user-images.githubusercontent.com/7273940/105152284-41b1c480-5b07-11eb-94fb-8b8e90660318.png">

Yes, it is how the data is.